pkg/controller/servingcert/starter/starter.go

```diff
@@ -64,3 +77,29 @@ func StartServiceServingCertSigner(ctx *controllercmd.ControllerContext) error {
 
 	return fmt.Errorf("stopped")
 }
+
+// readIntermediateCACert attempts to read an intermediate certificate
+// from the provided filename. If the file is missing, it is assumed
+// that no certificate was specified.
+func readIntermediateCACert(filename string) (*x509.Certificate, error) {
+	if len(filename) == 0 {
+		return nil, errors.New("filename not provided")
+	}
+
+	certsPEMBlock, err := ioutil.ReadFile(filename)
+	if os.IsNotExist(err) {
+		klog.V(4).Infof("%q does not exist which indicates that an intermediate certificate was not specified", filename)
+		return nil, nil
```

Comment:
shouldn't we fall over instead? When can this happen?

Reply:
An intermediate cert will only exist if rotation has performed. For a new cluster, that will not be true. I've updated the godoc for the function to reflect this, PTAL.